iothub/device/samples/getting started/EdgeModuleMethodSample/EdgeModuleMethodSample.cs

```diff
@@ -0,0 +1,106 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Diagnostics;
+using System.Text;
+
+namespace Microsoft.Azure.Devices.Client.Samples
+{
+    /// <summary>
+    /// This sample demonstrates how to send and receive direct methods on an Azure Edge module.
+    /// </summary>
+    public class EdgeModuleMethodSample
+    {
+        private readonly IotHubModuleClient _moduleClient;
+        private string _deviceId;
```

Comment:
Why are we not make deviceId and moduleId configurable?

Reply:
Trying to avoid asking more than necessary parameters from the user.